sdk/search/Azure.Search.Documents/src/Indexes/FieldBuilder.cs

```diff
@@ -229,7 +229,7 @@ ArgumentException FailOnUnknownDataType()
             return info.Properties
                 .Select(prop => (prop.Name, BuildField(prop)))
                 .Where(pair => pair.Item2 != null)
-                .ToDictionary(pair => pair.Item1, pair => pair.Item2);
+                .ToDictionary(pair => pair.Name, pair => pair.Item2);
```

Comment:
Why wasn't `Item2` updated?

Reply:
It's not required to name tuple fields when they are not part of the API surface but it's required that you use the named field when it exists.

Reply:
Right - IIRC, we shouldn't use tuples in public APIs anyway. But `Item2` has a name and it wasn't used here.